portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!--<title>My Portfolio - Sebastian Lopez</title>-->
```

Comment:
Is this code commented out? If so, please remove it completely. Same for other files.